README.md

```diff
@@ -88,6 +88,7 @@ Key:
 | v22.11                   | v1.0+           | v1.22+             |
 | v23.5                    | v1.0+           | v1.22+             |
 | v24.5                    | v1.0+           | v1.24+             |
+| v25.5                    | v1.0+           | v1.24+             |
```

Comment:
I'm not sure we want to include v25.5 here, as it does not exist at the moment. The same goes for `SECURITY.md` and `docs/documentation/pages/administrator/001-intro_administrator.cheatmd`